.gitignore

```diff
@@ -36,3 +37,7 @@ web-report/
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+**/.expo
+**/.expo-shared
+noode_modules
```

Comment:
Typo, please remove. You can find node modules being ignored in the first line of the file